pyanaconda/ui/gui/spokes/lib/cart.py

```diff
@@ -217,3 +235,26 @@ def on_set_as_boot_clicked(self, button):
             self._previousID = self._store[itr][ID_COL]
 
         self._toggle_button_text(self._store[itr])
+
+    def _initialize_secure_boot(self):
+        if not arch.is_s390():
+            self._secure_boot_box.hide()
```

Comment:
So if I understand this correctly, this is "just" a s390 variant of Secure Boot & is not adding support for "regular" UEFI Secure Boot? In such case I think this needs to be explicitly noted in the pull request that this is s390 specific and all the relevant boot options, kickstart options & GUI elements should be appropriately namespaced. 

Otherwise this could be easily confused with what most people know under Secure Boot (eq. UEFI Secure Boot) & could conflict with attempts to support UEFI Secure Boot in the future as well.

Reply:
Fixed. It is called ZIPL Secure Boot now.